pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -84,12 +96,18 @@ class TaskSerializer(ModelSerializer):
         read_only=True
     )
 
+    created_resources = CreatedResourceSerializer(
+        help_text=_('Resources created by this task.'),
```

Comment:
Are the resources actually created by the time the task returns? If not, these urls are just predictions right?